src/components/header/balance-card.tsx

```diff
@@ -1,39 +1,51 @@
 import { PersonTotalTime } from "../../generated/client";
 import { getHoursAndMinutes } from "../../utils/time-utils";
 import { Box, Grid, Typography } from "@mui/material";
-import Logo from "../../../img/Metatavu-icon.svg";
+import Logo from "../../../resources/img/Metatavu-icon.svg";
 import { userProfileAtom } from "../../atoms/auth";
 import { useAtomValue } from "jotai";
 
-interface BalanceCardProps {personTotalTime: PersonTotalTime | undefined}
+/**
+ * Component props
+ * @date 9/15/2023 - 11:15:09 AM
+ *
+ * @interface BalanceCardProps
+ * @typedef {BalanceCardProps}
+ */
+interface BalanceCardProps {
```

Comment:
As discussed yesterday in this context where we are defining the props in the same file as a component, they would typically just be called Props.